src/tex.js

```diff
@@ -57,6 +57,34 @@ MathJax.Extension.Arabic = {
       '8': '٨',
       '9': '٩'
     },
+    // numbers with arabic decimal separator ','
+    arabicDecimalSeparator: {
+      '0.': '\u0660\u066b',
+      '1.': '\u0661\u066b',
+      '2.': '\u0662\u066b',
+      '3.': '\u0663\u066b',
+      '4.': '\u0664\u066b',
+      '5.': '\u0665\u066b',
+      '6.': '\u0666\u066b',
+      '7.': '\u0667\u066b',
+      '8.': '\u0668\u066b',
+      '9.': '\u0669\u066b',
```

Comment:
@SamirElkhatib please use literal values like `٩` instead of `\u0669`. There's a build step to do that conversion. Let's keep the work human friendly.